### DISCUSSION TITLE
Rate limitation

### DISCUSSION THREAD

**Discussion by chendelbari** [community/community#128444] [npm]
Hi, is there any way to ramp up the rate limitation on querying the registry? 
Using additional api keys, by paid program, of anything else. 
Thanks

**Comment by MylesBorins** [comment]
@chendelbari can you share a bit more about which API you are hitting and why you need a rate limit increase? As the public registry is a shared resource "paid programs" are not something we would explore, but if current rate limits are insufficient we could explore raising them broadly. Alternatively there might be a more efficient way to accomplish what you are trying to do
